sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryOptions.java

```diff
@@ -53,4 +53,16 @@
   Integer getInsertBundleParallelism();
 
   void setInsertBundleParallelism(Integer parallelism);
+
+  @Description("The number of buckets used per table when doing streaming inserts to BigQuery.")
+  @Default.Integer(50)
+  Integer getNumStreamingBuckets();
```

Comment:
I think it is failing because you have to write the setter boilerplate.

Reply:
fixed